src/Plugins/ComplexCore/src/ComplexCore/Filters/ExtractVertexGeometryFilter.cpp

```diff
@@ -104,12 +104,12 @@ IFilter::PreflightResult ExtractVertexGeometryFilter::preflightImpl(const DataSt
   usize geomElementCount = dims[0] * dims[1] * dims[2];
   if(pUseMaskValue)
   {
-    const BoolArray& maskArray = dataStructure.getDataRefAs<BoolArray>(pMaskArrayPathValue);
-    if(maskArray.getNumberOfTuples() != geomElementCount)
+    const BoolArray* maskArray = dataStructure.getDataAs<BoolArray>(pMaskArrayPathValue);
+    if(maskArray->getNumberOfTuples() != geomElementCount)
```

Comment:
Use getDataRefAs, not getDataAs.

Reply:
I am actually fixing that entire filter on another branch. It is completely broken anyways.